pkg/dockerregistry/server/util.go

```diff
@@ -207,7 +209,82 @@ func (g *cachedImageStreamGetter) get() (*imageapi.ImageStream, error) {
 	return is, nil
 }
 
-func (g *cachedImageStreamGetter) cacheImageStream(is *imageapi.ImageStream) {
+func (g *cachedImageStreamGetter) cacheImageStream(is *imageapiv1.ImageStream) {
 	context.GetLogger(g.ctx).Debugf("(*cachedImageStreamGetter).cacheImageStream: got image stream %s/%s", is.Namespace, is.Name)
 	g.cachedImageStream = is
 }
+
+// resolveImageID returns latest TagEvent for specified imageID and an error if
+// there's more than one image matching the ID or when one does not exist.
```

Comment:
This is bad, and I don't think we should do it.  We want pkg/image/apis/image/helpers/v1 or something.